hack/release-build.sh

```diff
@@ -235,14 +235,17 @@ build_yamls() {
     pushd ${DIR}/charts
     find . -iname *.~?~ | xargs -r rm
 
+    releaseName=fission-$(echo ${version} | sed 's/\./-/g')
+    
     for c in fission-all fission-core
     do
-        # for minikube and other environment not support ELB
-        helm template ${c} -n ${c}-${version} --set serviceType=NodePort,routerServiceType=NodePort > ${c}-${version}-minikube.yaml
-        # for environments support ELB
-        helm template ${c} -n ${c}-${version} > ${c}-${version}.yaml
+        # for minikube and other environment that don't support LoadBalancer
+        helm template ${c} -n ${releaseName} --set serviceType=NodePort,routerServiceType=NodePort > ${c}-${version}-minikube.yaml
+        # for environments that support LoadBalancer
+        helm template ${c} -n ${releaseName} > ${c}-${version}.yaml
         # for cases where prometheus installation along with fission is not preferred
-        helm template ${c} -n ${c}-${version} --set canaryDeployment.prometheusDeploy=false > ${c}-${version}-fission-only.yaml
+        helm template ${c} -n ${releaseName} --set canaryDeployment.prometheusDeploy=false > ${c}-${version}-fission-only.yaml
```

Comment:
actually we moved prometheusDeploy flag out of canaryDeployment config. I can change this or you could, do let me know.

Reply:
I'll just remove this yaml for now.  People can just choose fission-core for the minimal install or fission-all for the all-inclusive one.  We can never really cover all the possible flags in these yamls.

Reply:
Ok, let me make a PR with prometheusDeploy = false in values.yaml for fission-core.